lib/mi_bridges/driver.rb

```diff
@@ -66,25 +68,33 @@ def initialize(snap_application:, logger: nil)
     def run
       setup
 
-      page_classes = SIGN_UP_FLOW + APPLY_FLOW
-
-      page_classes.each do |klass|
+      SIGN_UP_FLOW.each do |klass|
         begin
           page = klass.new(@snap_application, logger: logger)
           page.setup
           page.fill_in_required_fields
           page.continue
         rescue StandardError => e
-          # rubocop:disable Debugger
-          binding.pry if ENV["DEBUG_DRIVE"]
-          # rubocop:enable Debugger
-          throw e
+          debug(e)
         end
       end
 
+      complete_flow_steps
```

Comment:
Above we iterate through the sign up flow, then we while loop through the rest of the pages - is that right?

What do you think about breaking out the iterator above to its own method so it maintains a familiar pattern of implementation as `complete_flow_steps`. I mention that because I was having a hard time determining the difference between what goes on in the `each` and what gets tucked away in `complete_flow_steps`. The former being implemented in the body of this method, versus the latter in a private method threw me off a little bit.

something like

```
sign_up_for_account
complete_application
```

Reply:
great suggestion! done at e7be38d